build-all.sh

```diff
@@ -22,6 +22,15 @@ if [ $# -lt 1 ]; then
 fi
 PREFIX="$1"
 
+installed() {
+  if ! hash $1 2>/dev/null; then
+    echo "$1 is not installed. Please install it and retry" 1>&2
+    exit 1
+  fi
+}
+
+installed git
+installed svn
```

Comment:
Looks sensible, but a few nits:

* I'm using 4 space indentation in the shell scripts
* I use the (redundant) `if [ condition ]; then` style for conditionals

Also an empty line after the installed commands would be nice. And maybe checking for cmake as well?

I wasn't familiar with the `hash` shell builtin, I use `if [ -n "$(which cmd)" ]; then` in a few existing places though - I guess `hash`is better, but there's no need to rush to fix those others for consistency. (`which` in msys has the annoying habit of printing to stderr on error, contrary to the one usually used on linux, and I haven't bothered to redirect stderr for those cases.)

Reply:
`if []` doesn't work with commands. `[` is an alias to `test`.

The rest is done.

Reply:
Ah, right yes.

Empty line between the checks and the `build-*` calls please?